rosbag2_storage_sqlite3/include/rosbag2_storage_sqlite3/sqlite_storage.hpp

```diff
@@ -125,13 +125,16 @@ class ROSBAG2_STORAGE_DEFAULT_PLUGINS_PUBLIC SqliteStorage
   RCPPUTILS_TSA_REQUIRES(database_write_mutex_);
   int get_last_rowid();
   int read_db_schema_version();
+  uint64_t get_page_size();
+  uint64_t read_total_page_count_locked() RCPPUTILS_TSA_GUARDED_BY(database_write_mutex_);
```

Comment:
This is marked as "GUARDED_BY" `database_write_mutex_`, but this isn't a write to the database, is it?  It seems like either the mutex is misnamed, or we need another mutex.

Also, this function is logically "const", though I don't know if it can be that for the same reasons as `get_page_size`.

Reply:
Yes, probably the name of the mutex is not accurate. The situation is that we can't write and read from the database concurrently. Only one SQL statement is allowed per time. I am guessing that the name originally was chosen as `write` mutex because we didn't have a situation before when it was possible to call DB read and write operations from multiple threads.  
However, I can rename it to the `db_read_write_mutex`.